crypten/models/__init__.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import importlib.util
+import sys
+
+import crypten.nn as cnn
+import torch
+
+
+# List of modules to import and additional classes to update from them
+__import_list = {
+    "alexnet": [],
+    "densenet": ["_DenseLayer", "_DenseBlock", "_Transition"],
+    "googlenet": ["Inception", "InceptionAux", "BasicConv2d"],
+    "inception": [
+        "BasicConv2d",
+        "InceptionA",
+        "InceptionB",
+        "InceptionC",
+        "InceptionD",
+        "InceptionE",
+        "InceptionAux",
+    ],
+    "mnasnet": ["_InvertedResidual"],
+    "mobilenet": ["ConvBNReLU", "InvertedResidual"],
+    "resnet": ["BasicBlock", "Bottleneck"],
+    "shufflenetv2": ["InvertedResidual"],
+    "squeezenet": ["Fire"],
+    "vgg": [],
+}
+
+
+__all__ = []
+
+
+def __import_module_copy(module_name):
+    """
+    Returns a copy of an imported module so it can be modified
+    without modifying future imports of the given module
+    """
+    starting_modules = sys.modules.copy()
+
+    module_spec = importlib.util.find_spec(module_name)
+    module = importlib.util.module_from_spec(module_spec)
+    module_spec.loader.exec_module(module)
+    new_modules = set(sys.modules) - set(starting_modules)
+
+    del module_spec
```

Comment:
Q: Why this needs to be deleted? (wouldn't it get deleted when the function finishes?)